packages/contracts-bedrock/test/L1/OptimismPortalInterop.t.sol

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Testing utilities
+import { CommonTest } from "test/setup/CommonTest.sol";
+
+// Libraries
+import { Constants } from "src/libraries/Constants.sol";
+import { Predeploys } from "src/libraries/Predeploys.sol";
+
+// Target contract dependencies
+import "src/libraries/PortalErrors.sol";
+import { OptimismPortalInterop } from "src/L1/OptimismPortalInterop.sol";
+import { L1BlockInterop, ConfigType } from "src/L2/L1BlockInterop.sol";
+
+contract OptimismPortalInterop_Test is CommonTest {
+    /// @notice Marked virtual to be overridden in
+    ///         test/kontrol/deployment/DeploymentSummary.t.sol
+    function setUp() public virtual override {
+        super.enableInterop();
+        super.setUp();
+    }
+
+    /// @dev Tests that the config for the gas paying token can be set.
+    function testFuzz_setConfig_gasPayingToken_succeeds(bytes calldata _value) public {
```

Comment:
Another way to think about this would be to fuzz with realistic inputs ie have fuzz targets be the inputs to the encoding library and then pass along those bytes in the `setConfig` call

Reply:
I think it's ok with `bytes calldata _value` because `optimismPortalInterop` doesn't do any validation/decoding and just emits `_value`, which we check for, so it's ok to fuzz against `_value` being an opaque set of bytes. thoughts?